### DISCUSSION TITLE
Switch to main branch

### DISCUSSION THREAD

**Discussion by jtagcat** [archlinux/archinstall#433] [General]
```
    branches:
      - master
      - main # In case we adopt this convention in the future
```

It's easier to switch earlier than sooner, or sunset any changes at all.

**Comment by Torxed** [comment]
Where do you want this change to happen?

**Reply by jtagcat** [reply]
https://github.com/archlinux/archinstall you mean then?

I don't _want_ any change tbh, just if there's a plan to rename master to main, it's probably best done sooner than later.
